lib/spack/spack/test/build_environment.py

```diff
@@ -63,7 +63,8 @@ def build_environment(working_env):
     os.environ["SPACK_LINKER_ARG"] = "-Wl,"
     os.environ["SPACK_DTAGS_TO_ADD"] = "--disable-new-dtags"
     os.environ["SPACK_DTAGS_TO_STRIP"] = "--enable-new-dtags"
-    os.environ["SPACK_SYSTEM_DIRS"] = "/usr/include /usr/lib"
+    os.environ["SPACK_SYSTEM_DIRS"] = "/usr/include|/usr/lib"
+    os.environ["SPACK_MANAGED_DIRS"] = f"{prefix}/opt/spack"
```

Comment:
This should probably be set to the value of `install_tree:root` from the config.

Reply:
this is in a test fixture only used locally in unit tests here, no integration tests, so should be OK like this

Reply:
Got it, missed the context.  Thanks!